src/ppx/application.cpp

```diff
@@ -1438,6 +1440,10 @@ int Application::Run(int argc, char** argv)
                 break;
             }
 
+            if (mTogglePassthroughFrameCount > 0 && mFrameCount % mTogglePassthroughFrameCount == 0) {
+                mXrComponent.TogglePassthrough();
```

Comment:
What's the goal of toggling the passthrough every N frames?

Reply:
It was useful for testing the feature. Removed for now.